DICOM/src/Microsoft.Health.DeID.SharedLib/Models/AgeObject.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.DeID.SharedLib.Models
+{
+    public class AgeObject
```

Comment:
🟡 **SUGGESTION**
Maybe  `DicomAge` is better?

Reply:
Do we need to move this object to DICOM project?

Reply:
Does the SharedLib contain basic functions which can be shared by DICOM or FHIR?
If so, yes, DICOM.Anonymizer.Core is a better place for it. (along with the AgeType?)

Reply:
❓ [Question]
BTW, suppose the SharedLib will serve both DICOM and FHIR, why is the namespace called `Microsoft.Health.Dicom.DeID.SharedLib`?
The project name is **Microsoft.Health.DeID.SharedLib**, this makes sense.

Reply:
I think AgeType can be used by DICOM and FHIR?